apps/monitoring-cf.yml

```diff
@@ -0,0 +1,51 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Parameters:
+  StepFunctionArn:
+    Type: String
+
+  ApiId:
+    Type: String
+
+Resources:
+
+  AlarmTopic:
+    Type: AWS::SNS::Topic
+
+  StepFunctionAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      AlarmActions:
+        - !Ref AlarmTopic
+      AlarmDescription: Hyp3 processing failures
+      ComparisonOperator: GreaterThanThreshold
+      Period: 60
+      EvaluationPeriods: 1
+      Threshold: 0
+      Dimensions:
+        - Name: Processing Failures
+          Value: !Ref StepFunctionArn
+      MetricName: ExecutionsFailed
+      Namespace: AWS/States
+      Statistic: Sum
+      Unit: Count
+      TreatMissingData: notBreaching
+
+  ApiAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      AlarmActions:
+        - !Ref AlarmTopic
+      AlarmDescription: Hyp3 api HTTP 500 errors
+      ComparisonOperator: GreaterThanThreshold
+      Period: 60
+      EvaluationPeriods: 1
+      Threshold: 0
+      Dimensions:
+        - Name: API 500 errors
+          Value: !Ref ApiId
+      MetricName: 5xxError
```

Comment:
just `5xx` I believe

Reply:
https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-metrics-and-dimensions.html
I think the console displays a simplified version

Reply:
although this documention reveals that our demention isn't valid